packages/react-core/src/demos/examples/AlertGroup/AlertGroupDemo.tsx

```diff
@@ -0,0 +1,423 @@
+                </NotificationDrawerListItemHeader>
+                <NotificationDrawerListItemBody timestamp={timestamp}> {description} </NotificationDrawerListItemBody>
+              </NotificationDrawerListItem>
+            ))}
+          </NotificationDrawerList>
+        )}
+        {notifications.length === 0 && (
+          <EmptyState variant={EmptyStateVariant.full}>
+            <EmptyStateIcon icon={SearchIcon} />
+            <Title headingLevel="h2" size="lg">
+              No notifications found
+            </Title>
+            <EmptyStateBody>There are currently no notifications.</EmptyStateBody>
+          </EmptyState>
+        )}
+      </NotificationDrawerBody>
+    </NotificationDrawer>
+  );
+
+  return (
+    <DashboardWrapper
+      header={<DashboardHeader notificationBadge={notificationBadge} />}
+      mainContainerId="main-content-page-layout-default-nav"
+      notificationDrawer={notificationDrawer}
+      isNotificationDrawerExpanded={isDrawerExpanded}
+    >
+      <PageSection variant={PageSectionVariants.light}>
+        <TextContent>
+          <Text component="h1">Alert Group with Notification Drawer demo</Text>
+          <Text component="p">
+            New alerts can be added with buttons below. By default, only 3 alerts are displayed. <br />
+            The rest can be accessed in the Notification Drawer after clicking on the bell icon in the header
+            <br /> or by clicking on the overflow message.
+          </Text>
+        </TextContent>
+      </PageSection>
+
+      <PageSection variant={PageSectionVariants.light}>
+        <button
+          onClick={addSuccessAlertNotification}
+          type="button"
+          className={alertButtonClasses}
+          style={alertButtonStyle}
+        >
+          Add toast success alert
+        </button>
+        <button
+          onClick={addDangerAlertNotification}
+          type="button"
+          className={alertButtonClasses}
+          style={alertButtonStyle}
+        >
+          Add toast danger alert
+        </button>
+        <button
+          onClick={addInfoAlertNotification}
+          type="button"
+          className={alertButtonClasses}
+          style={alertButtonStyle}
+        >
+          Add toast info alert
+        </button>
+        <br />
+        <br />
+        <button
+          onClick={addWarningAlertNotification}
+          type="button"
+          className={alertButtonClasses}
+          style={alertButtonStyle}
+        >
+          Add toast warning alert
+        </button>
+        <button
+          onClick={addDefaultAlertNotification}
+          type="button"
+          className={alertButtonClasses}
+          style={alertButtonStyle}
+        >
+          Add toast default alert
+        </button>
+      </PageSection>
+
+      <PageSection variant={PageSectionVariants.light}>
+        <TextContent>
+          <br />
+          <Text component="h3">Max displayed alerts</Text>
```

Comment:
```suggestion
          <Text component="h2">Max displayed alerts</Text>
```

Since the previous heading used was an `h1`, we should use an `h2` here to avoid skipping heading levels.